public/expressions/helpers.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { getClient } from '../services';
+import {
+  VisLayerTypes,
+  VisLayerError,
+  VisLayerErrorTypes,
+  PointInTimeEventsVisLayer,
+} from '../../../../src/plugins/vis_augmenter/public';
+
+export const getAlerts = async (
+  monitorId: string,
+  startTime: number,
+  endTime: number
+): Promise<string> => {
+  const params = {
+    size: 1000,
+    sortField: 'start_time',
+    sortDirection: 'asc',
+    monitorIds: [monitorId],
+  };
+
+  const resp = await getClient().get('/api/alerting/alerts', { query: params });
+
+  if (resp.ok) {
+    const filteredAlerts = resp.alerts.filter((alert) => alert.start_time >= startTime && alert.start_time <= endTime );
+    return filteredAlerts;
+  } else {
+    console.log('error getting alerts:', resp);
```

Comment:
Nit: This won't help us, so we can remove it. If you want to keep it, I would use console.error instead

Reply:
Changed to `console.error`